api/notd/gallery_manager.py

```diff
@@ -333,3 +333,9 @@ async def follow_gallery_user(self, registryAddress: str, userAddress: str, acco
         if not userProfile.twitterId:
             raise BadRequestException('NO_TWITTER_ID')
         await self.twitterManager.follow_user_from_user(userTwitterId=accountProfile.twitterId, targetTwitterId=userProfile.twitterId)
+
+    async def update_all_twitter_users_deferred(self) -> None:
```

Comment:
would be better to put these functions in the regular manager (and add twitterManager as a dependency to that)